<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>libraryWindow</class>
 <widget class="QDialog" name="libraryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-color:rgb(33, 33, 33);
}</string>
  </property>
  <widget class="QLabel" name="pageLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color: white;
}</string>
   </property>
   <property name="text">
    <string>Your Library</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="exitLibButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>430</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Baskerville</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Exit Library</string>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Baskerville</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
background-color:rgb(57, 83, 141);
color: rgb(255, 255, 255)
}</string>
   </property>
   <property name="text">
    <string>Book Title</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Baskerville</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
background-color:rgb(57, 83, 141);
color: rgb(255, 255, 255)
}</string>
   </property>
   <property name="text">
    <string>Book Author</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="titleLineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="authorLineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Baskerville</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:rgb(57, 83, 141);
color: rgb(255, 255, 255)
}</string>
   </property>
   <property name="text">
    <string>Add Book</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Baskerville</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Baskerville</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color: white;
}</string>
   </property>
   <property name="text">
    <string>List of Your Books</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
